docs/source/markdown/podmansh.1.md

```diff
@@ -19,20 +19,22 @@ The user is confined to the container environment via all of the security mechan
 Systemd will automatically create the container when the user session is started. Systemd will take down the container when all connections to the user session are removed. This means users can log in to the system multiple times, with each session connected to the same container.
 
 ## Setup
-Modify user login session using usermod
+Create user login session using useradd
 
 ```
-# usermod -s /usr/bin/podmansh testu
-# grep testu /etc/passwd
-testu:x:4004:4004::/home/testu:/usr/bin/podmansh
+# useradd -s /usr/bin/podmansh lockedu
+# grep lockedu /etc/passwd
+lockedu:x:4008:4008::/home/lockedu:/usr/bin/podmansh
 ```
 
 Now create a podman Quadlet file that looks something like one of the following.
 
 Fully locked down container, no access to host OS.
 
 ```
-sudo cat > /etc/containers/systemd/users/podmansh.container << _EOF
+UID=$(id -u lockedu)
+mkdir -p /etc/containers/systemd/users/${UID}
+sudo cat > /etc/containers/systemd/users/${UID}/podmansh.container << _EOF
```

Comment:
...and the `sudo` here suggests you are not. Pick one or the other. I super-strongly suggest picking root, because `sudo cat > /path/only/writable/by/root` does not work, you need to do the horrible `| tee` thing.

Suggestion: remove the `sudo`, and add an explicit "run these commands as root" to line 33-ish.